<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPage</class>
 <widget class="QWizardPage" name="GeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General options</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="title">
   <string>General options</string>
  </property>
  <property name="subTitle">
   <string>Select general options for validation and raports</string>
  </property>
  <attribute name="pageId">
   <string notr="true">0</string>
  </attribute>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="srcLabel">
     <property name="text">
      <string>Source directory</string>
     </property>
     <property name="buddy">
      <cstring>srcButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="srcLayout">
     <item>
      <widget class="QLineEdit" name="srcEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select root directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="srcButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="logLabel">
     <property name="text">
      <string>Log directory</string>
     </property>
     <property name="buddy">
      <cstring>logButton</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="logLayout">
     <item>
      <widget class="QLineEdit" name="logEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select log directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="logButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="policyLabel">
     <property name="text">
      <string>Actions</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="policyLayout">
     <item>
      <widget class="QCheckBox" name="logenCheck">
       <property name="text">
        <string>Enable raport creation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autorunCheck">
       <property name="text">
        <string>Automatic run after wizard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>srcEdit</tabstop>
  <tabstop>srcButton</tabstop>
  <tabstop>logEdit</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>logenCheck</tabstop>
  <tabstop>autorunCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
